pkg/apis/autoscaling/v2beta1/conversion.go

```diff
@@ -257,3 +259,45 @@ func Convert_v2beta1_PodsMetricStatus_To_autoscaling_PodsMetricStatus(in *autosc
 	}
 	return nil
 }
+
+func Convert_autoscaling_HorizontalPodAutoscaler_To_v2beta1_HorizontalPodAutoscaler(in *autoscaling.HorizontalPodAutoscaler, out *autoscalingv2beta1.HorizontalPodAutoscaler, s conversion.Scope) error {
+	if err := autoConvert_autoscaling_HorizontalPodAutoscaler_To_v2beta1_HorizontalPodAutoscaler(in, out, s); err != nil {
+		return err
+	}
+	if in.Spec.Behavior != nil {
+		old := out.Annotations
+		out.Annotations = make(map[string]string, len(old)+1)
+		for k, v := range old {
+			out.Annotations[k] = v
+		}
+
+		behaviorEnc, err := json.Marshal(in.Spec.Behavior)
+		if err != nil {
+			return err
+		}
```

Comment:
This entire block of code is redundant. There are other places where similar code is used but it really adds nothing.

Reply:
@arjunrn : what block do you mean? lines 275-277?